### DISCUSSION TITLE
Future of databasez

### DISCUSSION THREAD

**Discussion by EssaAlshammri** [dymmond/databasez#9] [Q&A] [answered]
Hi @tarsil thank you for your awesome work on databasez

I just have some questions.

Does this pull changes from encode/databases?

What will happen to databasez when encode/databases support sqlalchemy 2?

Do you have plans for databasez to go other directions in the future?

Currently I'm using databases and waiting for your pull request there to be finalized.
And I just removed databases in favor of databasez and all I did was alias databases = databasez
And everything worked great.

Thanks

**Comment by tarsil** [comment, marked as answer]
@EssaAlshammri thank you for your comment and opened discussion.

Encode has done a terrific (in a good way) job with databases, unfortunately they have no time allocated to maintaining it the way we would love to. I was blocked with the previous version and that's why I forked, fixed, updated and added features to it.

What is the future? I will always maintain Databasez 🙂. Even encode merging the PR, Databasez is already different from that anyway and it's also being used by Saffier (https://saffier.tarsild.io) and Esmerald (https://esmerald.dev) which makes me even more motivated to continue to maintaining it.

This is for the community so Databasez can grow the way it was meant to so suggestions, PRs, fixes and so on are more than welcomed 🙂

Databasez is also different from its origins already but 100% compatible with it as it was intentionally designed to.

Some new features are in the design process at the moment and being validated. Hopefully I can start the development soon.

Does this answer your question?
